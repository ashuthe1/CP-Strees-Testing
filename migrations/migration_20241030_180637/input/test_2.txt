30
-8 -2 3 5 -9 10 6 10 3 -2 -1 -4 -2 0 8 -9 2 0 -4 2 -1 -1 4 0 4 6 10 1 2 6 
86
47 258
115 444
103 221
111 390
228 390
373 461
396 460
189 380
91 211
23 160
53 56
252 343
147 168
101 104
153 358
71 222
49 405
42 149
270 350
309 430
274 425
28 80
167 408
269 377
33 291
348 365
155 225
151 322
66 184
148 423
305 424
175 182
330 354
216 435
180 214
22 90
118 416
102 117
60 182
305 369
7 187
161 268
224 230
295 407
107 252
65 366
82 91
239 256
267 411
37 159
127 258
209 458
244 371
115 137
87 442
295 370
97 440
198 342
138 171
244 416
317 422
121 325
359 398
115 156
152 437
230 314
90 306
250 333
211 386
109 259
88 174
13 426
184 355
133 158
60 386
294 376
31 237
153 446
79 202
136 173
42 262
131 402
186 379
275 402
68 384
45 369