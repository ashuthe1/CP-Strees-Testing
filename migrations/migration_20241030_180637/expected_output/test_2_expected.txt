3057
2824
1922
2244
502
1129
975
616
2137
2713
106
302
538
62
1177
2589
3649
2162
340
1012
936
1030
1234
480
3229
4
896
1039
2340
1900
900
73
37
1007
451
1374
2348
314
2373
325
3398
692
-30
663
1788
2989
246
-39
799
2479
1446
1515
323
288
3353
317
3134
334
826
751
783
1623
319
736
2077
92
2089
248
479
1783
1684
4479
572
580
3211
438
3347
2177
2311
900
3130
2012
713
635
3054
3395
